docs/guides/core-concepts/introduction-to-cypress.mdx

```diff
@@ -913,11 +913,10 @@ randomly fail. This would lead to flaky, inconsistent results.
 
 :::info
 
-While Cypress is built using Promises that come from
-[Bluebird](http://bluebirdjs.com/), these are not what we expose as commands and
-assertions on `cy`. If you'd like to learn more about handling asynchronous
-Cypress Commands please read our
-[Core Concept Guide](/guides/core-concepts/variables-and-aliases).
+While Cypress does have a [`.then()`](/api/commands/then) command, Cypress
+commands are not promises and cannot be `await`ed. If you'd like to learn more
```

Comment:
Can you update the first mention of the `.then()` command with the command link as well?
```suggestion
commands are not Promises and cannot be `await`ed. If you'd like to learn more
```